docker-compose.yml

```diff
@@ -26,6 +11,8 @@ services:
       POSTGRES_PASSWORD: secret
       POSTGRES_USER: developer
       POSTGRES_DB: store
-    volumes: 
-      - ./infra:/docker-entrypoint-initdb.d/
```

Comment:
ingen aning om vad detta är. hoppas det inte behövs iaf

Reply:
Tror denna rad säger "kör alla queries i alla .sql-filer i /infra", dvs initdb.sql i detta fallet. Skulle någon vilja snurra upp detta projekt nu måste man manuellt gå in och skapa upp tabellen/köra initdb.sql

Rent spontant skulle jag säga att den med fördel ligger kvar som den låg